ui/src/album/AlbumActions.js

```diff
@@ -78,20 +51,7 @@ const AlbumActions = ({
 
   return (
     <TopToolbar className={className} {...sanitizeListRestProps(rest)}>
-      <PlayButton
-        button={() => (
-          <Button
-            onClick={playing ? () => dispatch(pausePlayer()) : handlePlay}
-            label={
-              playing
-                ? translate('resources.album.actions.pause')
-                : translate('resources.album.actions.playAll')
-            }
-          >
-            {playing ? <PauseIcon /> : <PlayArrowIcon />}
-          </Button>
-        )}
-      />
+      <PlayButton record={record} buttonType="button" handlePlay={handlePlay} />
```

Comment:
I was unsure about how to instantiate a component passed down as a prop in PlayButton.js, so I passed it as a string and conditionally rendered Button or IconButton

Reply:
Did you take a look at the examples I posted? They do exactly this, pass a component as a property to another

Reply:
I did take a look but both the components were React Admin components so I couldn't figure out what was being done behind the scenes to instantiate it the way I need it to be done. If you could point me in a direction, I can change it up to the way that's a better practice.

Reply:
Have you looked at React Admin (or Material-UI) code?  You can have really good insights looking there.

Another example (this time in our code) is `common/SimpleList` that can receive a component in the `leftIcon` and `leftAvatar` properties.

Reply:
Got it! Thanks a lot. I will make these changes and push right away.